networking/network_observability/network-observability-overview.adoc

```diff
@@ -36,6 +36,7 @@ The Network Observability Operator provides the Flow Collector API custom resour
 
 The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
 
+Additionally, in *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overview of the persistent contents of your network. You can view metrics, such as *Top 5 flow rates per minute per namespace*, *Top 5 Ingress bandwidth per source and destination nodes*, and *Top 5 Ingress bandwidth per source and destination namespace*. Also in *Observe* -> *Dashboards*, the *Netobserv/Health* dashboard provides metrics about the health of the Operator.
```

Comment:
oh by the way, I'm renaming these metrics [here](https://github.com/netobserv/network-observability-operator/pull/356/files#diff-465b40370876d30fb6347584d1738536c64758d1be0d572481b9b979f6392f6c) so it should be renamed here as well, the previous names were not 100% correct. We now have by default:
- Top flow rates per source and destination namespaces
- Top byte rates received per source and destination nodes
- Top byte rates received per source and destination workloads

(this PR is still under review, you can comment about these titles)

Reply:
@jotak I took another look at this PR now that its been merged, and noticed that now there are the following:
* Top byte rates emitted per source and destination nodes/workloads
* Top packet rates emitted/received per source and destination workloads

I added these to a the list in the documentation. Can you PTAL and verify that these additions are OK?

Reply:
yeah that's fine, I didn't mention them because I thought it was on purpose that you didn't list everything (there's the "`such as`...")

Reply:
Ahh ok I see. NP, I added them all :)